ddp-workspace/projects/ddp-sdk/src/lib/directives/scroll-up.directive.ts

```diff
@@ -0,0 +1,27 @@
+import { Directive, ElementRef, EventEmitter, Inject, Input, OnChanges, OnInit, Output, SimpleChanges } from "@angular/core";
+import { ConfigurationService } from "../services/configuration.service";
+import { WindowRef } from "../services/windowRef";
+
+@Directive({
+    selector: '[scroll-up]'
+})
+export class ScrollUpDirective implements OnChanges {
```

Comment:
I've created a new directive in order to make it reusable since there are some components that are using the scroll-up functionality.

Reply:
In general. it is a good idea.
Why would not we use it in `ddp-sdk\src\lib\components\activityForm\activity-blocks\activityQuestion.component.ts` as well?

Reply:
done